var/spack/repos/builtin/packages/root/package.py

```diff
@@ -415,6 +419,10 @@ def cmake_args(self):
             options.append('-DCMAKE_PROGRAM_PATH={0}'.format(
                 self.spec['mysql-client'].prefix.bin))
 
+        if 'zstd' in self.spec:
+            options.append('-DZSTD_INCLUDE_DIR={0}'.format(
+                self.spec['zstd'].prefix.include))
```

Comment:
Just as a sanity check, did you make sure that zstd needs this hack before adding it?

Reply:
Thanks for asking!

I thought, it didn't work without these cmake defines.
But it seems to work without. So I removed it (check updated commit).

Reply:
Can we mark this resolved?

Reply:
I would, but the resolve button doesn't appear for me for some strange reasons :(

Reply:
Okay, I'll mark this resolved then
Thanks!